firefox-ios/Client/Frontend/Library/Bookmarks/BookmarksRefactorFeatureFlagProvider.swift

```diff
@@ -7,7 +7,11 @@ import Foundation
 protocol BookmarksRefactorFeatureFlagProvider {}
 
 extension BookmarksRefactorFeatureFlagProvider {
+    var featureFlags: LegacyFeatureFlagsManager {
+        return LegacyFeatureFlagsManager.shared
+    }
+
     var isBookmarkRefactorEnabled: Bool {
-        NimbusManager.shared.featureFlagLayer.checkNimbusConfigFor(.bookmarksRefactor)
+        return featureFlags.isFeatureEnabled(.bookmarksRefactor, checking: .buildOnly)
```

Comment:
@FilippoZazzeroni I didn't notice this on your first PR but when I was implementing this I realized that `isFeatureEnabled` is the correct method to use when we're checking the feature flags, otherwise it doesn't return the correct updated value when toggling the switch 🤔 just thought I'd mention it for next time!

Reply:
Ok perfect thank you @dataports. Then i'll know it for next features.